docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -852,16 +852,42 @@ multiFuzzyMatchAllIndices(haystack, distance, \[pattern<sub>1</sub>, pattern<sub
 
 ## extract
 
-Extracts a fragment of a string using a regular expression. If `haystack` does not match the `pattern` regex, an empty string is returned.
-
-For regex without subpatterns, the function uses the fragment that matches the entire regex. Otherwise, it uses the fragment that matches the first subpattern.
+Extracts a fragment of a string using a regular expression.
+If `haystack` does not match the `pattern` regex, an empty string is returned. 
+For a regex without sub-patterns, the function uses the fragment that matches the entire regex. 
+Otherwise, it uses the fragment that matches the first subpattern.
 
 **Syntax**
 
 ```sql
 extract(haystack, pattern)
 ```
 
+*Arguments**
+
+- `haystack` — Input string. [String](../data-types/string.md).
+- `pattern` — Regular expression with [re2 syntax](https://github.com/google/re2/wiki/Syntax).
```

Comment:
`with` --> `in`

Reply:
While at it, could you please replace in this file all occurrences of `re2 regular syntax` and `re2 syntax` (which sound weird) by `re2 regular expression syntax`?